hooks/useLogin.ts

```diff
@@ -36,7 +43,9 @@ export function useLogin(config: LoginProps = {}) {
       try {
```

Comment:
This isn't causing any user-perceivable issues, but we're making too many API calls because this function fires every time `$status` changes. On master it makes one call initially when status is `loading`, one call when status is `loaded`, and potentially an additional call when status is `forced` if the timeout happens.

Reply:
I added a condition here: https://github.com/iron-fish/website-testnet/pull/126/files#diff-787a09031878f88df5cd5bccb34bded89f4643ec5a089b98a9cfbff465e93019R46 which should prevent successive re-renders